components/Pricing.tsx

```diff
@@ -64,46 +75,140 @@ export default function Pricing({ products }: Props) {
             .
           </p>
         </div>
+        <LogoCloud />
+      </section>
+    );
+
+  if (products.length === 1)
+    return (
+      <section className="bg-black">
+        <div className="max-w-6xl px-4 py-8 mx-auto sm:py-24 sm:px-6 lg:px-8">
+          <div className="sm:flex sm:flex-col sm:align-center">
+            <h1 className="text-4xl font-extrabold text-white sm:text-center sm:text-6xl">
+              Pricing Plans
+            </h1>
+            <p className="max-w-2xl m-auto mt-5 text-xl text-zinc-200 sm:text-center sm:text-2xl">
+              Start building for free, then add a site plan to go live. Account
+              plans unlock additional features.
+            </p>
+            <div className="relative flex self-center mt-12 border rounded-lg bg-zinc-900 border-zinc-800">
+              <div className="border border-pink-500 border-opacity-50 divide-y rounded-lg shadow-sm bg-zinc-900 divide-zinc-600">
+                <div className="p-6 py-2 m-1 text-2xl font-medium text-white rounded-md shadow-sm border-zinc-800 whitespace-nowrap focus:outline-none focus:ring-2 focus:ring-pink-500 focus:ring-opacity-50 focus:z-10 sm:w-auto sm:px-8">
+                  {products[0].name}
+                </div>
+              </div>
+            </div>
+            <div className="mt-6 space-y-4 sm:mt-12 sm:space-y-0 sm:grid sm:grid-cols-2 sm:gap-6 lg:max-w-4xl lg:mx-auto xl:max-w-none xl:mx-0 xl:grid-cols-3">
+              {products[0].prices?.map((price) => {
+                const priceString =
+                  price.unit_amount &&
+                  new Intl.NumberFormat('en-US', {
+                    style: 'currency',
+                    currency: price.currency,
+                    minimumFractionDigits: 0
+                  }).format(price.unit_amount / 100);
+
+                return (
+                  <div
+                    key={price.interval}
+                    className={cn(
+                      'rounded-lg shadow-sm divide-y divide-zinc-600 bg-zinc-900',
+                      {
+                        'border border-pink-500': subscription
```

Comment:
the pink border isn't working for me, any idea why?

Reply:
Looks like there was an issue with how the subscription was being return, should work now